### PR TITLE
fixing bazel testing for upup

### PR DESCRIPTION
**Pull request by chrislovecnm**
Fixes: https://github.com/kubernetes/kops/issues/3626

### CODE REVIEW

**Comment by chrislovecnm** [comment]
/assign @justinsb 

@gambol99 @andrewsykim @KashifSaadat any of you guys know bazel?  Trying to figure out if someone else can review these other than just @justinsb

File: upup/pkg/fi/cloudup/BUILD.bazel [hunk 1]
```diff
@@ -103,4 +103,7 @@ go_test(
         "//vendor/github.com/golang/glog:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/util/sets:go_default_library",
     ],
+    data = [
```
Comment (andrewsykim) [inline upup/pkg/fi/cloudup/BUILD.bazel]: These were autogenerated right?

Comment (chrislovecnm) [reply upup/pkg/fi/cloudup/BUILD.bazel]: Testing stuff no. Gazelle will fix dependencies and stuff.

**Comment by justinsb** [comment]
/lgtm

Thanks for fixing!

**Pull request merged**